### DISCUSSION TITLE
How is the % muriatic acid used?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#753] [REM] [answered]
![image](https://user-images.githubusercontent.com/6830886/236053319-fc4e5324-53a6-4938-a85d-4a317dc975bb.png)

**Comment by tagyoureit** [comment, marked as answer]
In order to calculate how much acid to add to the pool, we need to know what concentration the acid is.  You can buy different % acids.  For example, the [Home Depot HDX](https://www.homedepot.com/p/HDX-1-Gal-Muriatic-Acid-Balancer-2-Pack-10031HDX/206809317) is 31.45%.  If you buy a weaker acid or dilute it, then it won't be as strong and we will need to dispense more to get to the target pH.

**Comment by rstrouse** [comment]
@tagyoureit beat me to the answer while I was typing out a dissertation.

It is used to calculate demand.  The molar concentration of acid determines how much needs to be dosed.  31.45% or 20 Baume is full strength muriatic acid.  At above 34.6% it is considered hydrochloric acid.   If you buy the 14.5% or 15.7% MA from the orange or blue store you are effectively diluting to 50%.  You DO NOT need to add water to that despite the mis-information out there about dilution.  

This all stems from folks not understanding what the term `full-strength` means wrt muriatic acid.  I suggest you simply shake your head and watch the armchair chemists pontificate about preserving equipment nonsense.  Even at full-strength HDPE and PVC plastics are practically immune to MA.  That is why it is shipped in such containers.

If you are using the Pentair tank simply remove the screw cap without removing the seal and invert it into the canister.   Then tap the top of it to break the seal.  If you are impatient like me poke a hole in the bottom of the jug after you break the seal and it will fill your tank in a matter of seconds without any exposure to the acid within.

**Comment by davthomaspilot** [comment]
I didn't realize the dose amount varied.  I figured it would be a constant dosage amount but that the time between doses varied.  If one wanted to get fancy, a PID algorithm could be implemented, I suppose.

Thanks.

**Comment by davthomaspilot** [comment]
If there are two "bodies", in my case spa and pool.

Is REM smart enough to associate ph and ORP readings based on which pump is running?  And, adjust volume or time of dose based on the what body is being pumped?

**Reply by rstrouse** [reply]
Simply put yes but only for volume dosing.  Time dosing exists but it is not the preferred option.  The demand is also recalculated during the dosing and if you have a really slow pump it will stop if the setpoint is reached.  But if it were me I would not dose when in spa mode there just really isn't any need.  

Simply set REM Chem to pool not pool/spa.  If you run a dual equipment pool and spa then it does make sense to operate separately and in that case you will have two REM Chem devices because they do not share any water.

**Reply by davthomaspilot** [reply]
"Simply set REM Chem to pool not pool/spa"

Missed that--yes, I agree, no need to dose when in spa mode.
